src/Definition.php

```diff
@@ -89,4 +89,11 @@ class Definition
      * @var string
      */
     public $documentation;
+
+    /**
+     * Parameters array (for methods and functions), for use in textDocument/signatureHelp
+     *
+     * @var string[]
```

Comment:
Yes, I currently populate only the label (using this string). I will add the documentation and construct `ParameterInformation` objects - of course that would mean an object for every parameter of every method / function in the project. I will test and let you know how it impacts memory usage.